contracts/nois-drand/src/contract.rs

```diff
@@ -456,11 +456,7 @@ fn execute_set_config(
     let config: Config = CONFIG.load(deps.storage)?;
 
     // check the calling address is the authorised multisig
-    ensure_eq!(
-        info.sender,
-        CONFIG.load(deps.storage)?.manager,
-        ContractError::Unauthorized
-    );
+    ensure_eq!(info.sender, config.manager, ContractError::Unauthorized);
```

Comment:
I did those changes in https://github.com/noislabs/nois-contracts/pull/195 separately